bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/ParallacticAngleControls.scala

```diff
@@ -194,7 +197,6 @@ class ParallacticAngleControls extends GridBagPanel with Publisher {
     for {
       e              <- editor
       ispObservation <- Option(e.getContextObservation)
-      if ObsTargetCalculatorService.targetCalculation(ispObservation).isDefined
```

Comment:
In the cases where we are doing the PA calculation the site is known, but in the general case where we're just editing the scheduling block it may not be. So this filter was preventing things from working. This is all very bad and I'm not helping matters much, but I don't have the spiritual energy to untangle this.